spk/sabnzbd/src/requirements.txt

```diff
@@ -1 +1,16 @@
-sabyenc==3.3.5
```

Comment:
May you please document "requirements.txt" generation in Makefile to ease future update.